src/app/services/board-item-control.service.ts

```diff
@@ -138,17 +145,23 @@ export class BoardItemControlService {
     });
   }
 
-  hideNameTags(isHidden: boolean): void {
+  // public function to correctly behave with both key and overriding Setting
+  showNameTags(isShown: boolean): void {
+    if (!this.gses.persistentNamePlates.value) this._showNameTags(isShown);
+  }
+
+  // private function to actually handle show/hide nametags
+  private _showNameTags(isShown: boolean): void {
```

Comment:
Underscore wird in Plain JS benutzt um private values von normalen zu unterscheiden. (Gibt dort kein private)
Da wir ja variablen und Funktionen als private deklarieren können brauchen wir das _ nicht.